frontend/tests/controllers/SchoolOfTheMonthTest.php

```diff
@@ -7,7 +7,13 @@
  * @author carlosabcs
  */
 class SchoolOfTheMonthTest extends \OmegaUp\Test\ControllerTestCase {
-    private static function setUpSchoolsRuns($schoolsData, $runDate = null) {
+    /**
+     * @param $schoolsData list<array{creator: \OmegaUp\DAO\VO\Identities, request: \OmegaUp\Request, response: array{status: string, school_id: int}, school: \OmegaUp\DAO\VO\Schools}>
+     */
+    private static function setUpSchoolsRuns(
+        array $schoolsData,
+        string $runDate = null
```

Comment:
nit:

```suggestion
        ?string $runDate = null
```

si el default es `null`, hay que hacer el tipo nullable.

Reply:
Done